connect/api/v1/recent_activity/views.py

```diff
@@ -15,7 +15,23 @@ class RecentActivityAPIView(views.APIView):
 
     def post(self, request):
 
-        if request.data.get("flow_organization"):
+        action = request.data.get("action")
+        entity = request.data.get("entity")
+        entity_name = request.data.get("entity_name")
+        user = User.objects.get(email=request.data.get("user"))
+
+        if(request.data.get("intelligence_id")):
+            organization = Organization.objects.get(inteligence_organization=request.data.get("intelligence_id"))
+            for project in organization.project.all():
+                RecentActivity.objects.create(
```

Comment:
As the amount of projects is unlimited, it would be better to do a bulk_create in RecentActivity.